proc/proc_linux.go

```diff
@@ -270,6 +272,10 @@ func (dbp *Process) trapWait(pid int) (*Thread, error) {
 			}
 			continue
 		}
+		if th == nil {
+			// Sometimes we get an unknown thread, ignore it?
```

Comment:
This doesn't seem right. The only time we should get an unknown thread is when a new one is cloned and it's the first time we're hearing about it. Was this crashing?